cves/kernel/CVE-2021-3573.yml

```diff
@@ -266,8 +276,8 @@ sandbox:
     Answer should be true or false
     Write a note about how you came to the conclusions you did, regardless of
     what your answer was.
-  answer:
-  note:
+  answer: true
+  note: this causes a UAF
```

Comment:
What is a UAF

Reply:
abbreviation of use-after-free that should probably be removed